examples/how_to/get_started.py

```diff
@@ -196,50 +199,91 @@
 print("Units found by tridesclous:", sorting_TDC.get_unit_ids())
 print("Units found by spyking-circus2:", sorting_SC2.get_unit_ids())
 
-# If a sorter is not installed locally, we can also avoid installing it and run it anyways, using a container (Docker or Singularity). For example, let's run `Kilosort2` using Docker:
+# If a sorter is not installed locally, we can also avoid installing it and run it anyways, using a container (Docker or Singularity).
+# To do this, you will need to install Docker. More information [here](https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html?highlight=docker#running-sorters-in-docker-singularity-containers).
+# Let's run `Kilosort2` using Docker:
 
 sorting_KS2 = ss.run_sorter(sorter_name="kilosort2", recording=recording_preprocessed, docker_image=True, verbose=True)
 print(sorting_KS2)
 
-# SpikeInterface provides a efficient way to extract waveforms from paired recording/sorting objects.
-# The `extract_waveforms` function samples some spikes (by default `max_spikes_per_unit=500`)
-# for each unit, extracts their waveforms, and stores them to disk. These waveforms are helpful to compute the average waveform, or "template", for each unit and then to compute, for example, quality metrics.
+# For postprocessing SpikeInterface pairs recording and sorting objects into a `SortingAnalyzer` object.
+# The `SortingAnalyzer` can be loaded in memory or saved in a folder. Here, we save it in binary format.
+
+sa_TDC = si.create_sorting_analyzer(sorting_TDC, recording_preprocessed, format='binary_folder', folder='sa_TDC_binary')
```

Comment:
```suggestion
sa_TDC = si.create_sorting_analyzer(sorting=sorting_TDC, recording=recording_preprocessed, format='binary_folder', folder='sa_TDC_binary')
```
Everyone decided that for tutorials we would always use keyword arguments to make things as explicit as possible  moving forward :)

Reply:
This is alsot lost in some of the loading functions that have no arguments specified.